DSCResources/MSFT_xCluster/MSFT_xCluster.psm1

```diff
@@ -104,6 +104,8 @@ function Set-TargetResource
 
             New-Cluster -Name $Name -Node $env:COMPUTERNAME -StaticAddress $StaticIPAddress -NoStorage -Force
 
+            Start-Sleep -Seconds 60
```

Comment:
You could use 
While (!(Get-Cluster)){Start-Sleep 5}
This discussion is the only reason, why the Quorum Packet is still not merged. I think we should fast solve it.